src/test/java/study/ReflectionTest.java

```diff
@@ -0,0 +1,123 @@
+package study;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+
+public class ReflectionTest {
+
+    private static final Logger logger = LoggerFactory.getLogger(ReflectionTest.class);
+
+    @Test
+    @DisplayName("Car 객체 정보 가져오기")
+    void showClass() {
+        Class<Car> carClass = Car.class;
+        logger.info("[클래스 정보 출력]");
+        logger.debug("name=" + carClass.getName());
+        logger.debug("modifiers=" + carClass.getModifiers());
```

Comment:
`logger`를 잘 활용해주셨네요! 
첫주차 첫미션이라 `logger`를 사용하신 이유는 충분히 이해됩니다 😄 

하지만, 좀더 멋진 검증을 가진 테스트 코드로 만들어 보면 어떨까요?

```suggestion
        Class<Car> carClass = Car.class;
        
        assertEquals("Car", carClass.getSimpleName(), "클래스 이름이 일치하지 않습니다.");
        assertEquals("com.study.Car", carClass.getName(), "클래스 전체 이름이 일치하지 않습니다.");
```

예를들면 이렇게요!

그리고 실행하게되면, 예상되는 값과 실제값이 왜 달라서 테스트 콘솔창에 명확하게 보여집니다.
이제는 테스트 코드 실행순서를 로그를 보면서 이해하면 따라가는것이 아닌, 코드 변경 시 즉각적인 피드백을 받을 수 있어, 개발 사이클을 더욱 효율적으로 만들 수 있어요 😄 
<img width="678" alt="스크린샷 2024-10-11 오후 1 12 59" src="https://github.com/user-attachments/assets/cc50878c-29d0-428d-9cbc-0c77309d7923">